packages/flutter_tools/lib/src/ios/setup_xcodeproj.dart

```diff
@@ -171,11 +155,11 @@ final String _infoPlistInitialContents = '''
 	<key>CFBundleExecutable</key>
 	<string>Runner</string>
 	<key>CFBundleIdentifier</key>
-	<string>io.flutter.runner.Runner</string>
+	<string>com.example.{{projectName}}</string>
```

Comment:
Good catch. I will just grep by the executable name since that remains constant.